src/main/java/firstSimpleJavaTasks/ArrayEditing.java

```diff
@@ -1,11 +1,10 @@
-package FirstSimpleJavaTasks;/*
+package firstSimpleJavaTasks;/*
```

Comment:
Please see general package names standards
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Regarding separation of words, you may choose different strategy, for example, here is a google's style guide
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Reply:
package renamed according to naming convention